main/src/main/scala/sbt/internal/AbstractTaskProgress.scala

```diff
@@ -122,7 +122,7 @@ private[sbt] abstract class AbstractTaskExecuteProgress extends ExecuteProgress[
 object AbstractTaskExecuteProgress {
   private[sbt] class Timer() {
     val startNanos: Long = System.nanoTime()
-    val threadId: Long = Thread.currentThread().getId
+    val threadId: Long = Thread.currentThread().threadId
```

Comment:
`threadId` was added in JDK 19, so this won't work on JDK 8, 11, or 17?

Reply:
Oh! You're right: `@since 19`.

Dam, this won't work like that… :disappointed: